routes/web.php

```diff
@@ -17,48 +18,51 @@
 Route::post('logout')->name('logout')->uses('Auth\LoginController@logout');
 
 // Dashboard
-Route::get('/')->name('dashboard')->uses('DashboardController')->middleware('auth');
+Route::get('/')->name('dashboard')->uses('DashboardController');
 
-// Accounts
-Route::get('accounts')->name('accounts')->uses('AccountsController@index')->middleware('remember', 'auth');
-Route::get('accounts/create')->name('accounts.create')->uses('AccountsController@create')->middleware('auth');
-Route::post('accounts')->name('accounts.store')->uses('AccountsController@store')->middleware('auth');
-Route::get('accounts/{account}/edit')->name('accounts.edit')->uses('AccountsController@edit')->middleware('auth');
-Route::put('accounts/{account}')->name('accounts.update')->uses('AccountsController@update')->middleware('auth');
-Route::delete('accounts/{account}')->name('accounts.destroy')->uses('AccountsController@destroy')->middleware('auth');
-Route::put('accounts/{account}/restore')->name('accounts.restore')->uses('AccountsController@restore')->middleware('auth');
+Route::group(['middleware' => ['auth']],function(){
```

Comment:
I'd prefer to leave the routes as they were, without the the groups. Please revert that change.

Reply:
there is no issue with group routes and its does not change anything.i have done this because you have added `auth` middleware in except `login and logout` so i have grouped together to add the middleware and makes the code clean. 

If you dont like that i can 
**add the middleware in _construct method of controller** 
or
 **do you need to revert it to original??**

Reply:
I would prefer to leave it as it was.